x-pack/plugins/ml/public/application/components/stats_bar/stats_bar.tsx

```diff
@@ -37,9 +38,13 @@ interface StatsBarProps {
 }
 
 export const StatsBar: FC<StatsBarProps> = ({ stats, dataTestSub }) => {
+  const { euiTheme } = useEuiTheme();
   const statsList = Object.keys(stats).map((k) => stats[k as StatsKey]);
   return (
-    <div className="mlStatsBar" data-test-subj={dataTestSub}>
+    <div
+      css={{ height: '42px', padding: '14px', backgroundColor: euiTheme.colors.lightestShade }}
```

Comment:
do we need a hardcoded height? and can we set padding based on the euiTheme?

Reply:
Updated in https://github.com/elastic/kibana/pull/152240/commits/c0b3e32ad44dbd2646b2c2e627c3b9858d3c812b